XP-Pen_Deco_01/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "0042"
+vendor: Ugee
```

Comment:
The vendor here and in the directory name need to match. It was me probably who suggested to use "Ugee" here, but looking at the web, I cannot find a tablet named "Ugee Deco 01", so it wouldn't make much sense to name it as such. Therefore we need to put "XP-Pen" here.